client/app/components/dialog-dashboard/dialog-dashboard.component.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .component('dialogDashboard', {
+      controller: function(DialogEdit, lodash) {
+        /**
+         * Load component data
+         */
+        this.$onInit = function() {
+          this.service = DialogEdit;
+          // load data from service
+          this.dialogTabs = DialogEdit.getData().content[0].dialog_tabs;
```

Comment:
Seeing you're doing that `getData().content[0].dialog_tabs` in multiple places, maybe that's a bit of an implementation detail, that should be hidden away in the service.

Maybe even in general, might prefer to move more logic away from the controller into the service..